pkg/client/channel/invoke/txnhandler.go

```diff
@@ -63,7 +64,13 @@ func (e *EndorsementHandler) Handle(requestContext *RequestContext, clientContex
 
 	requestContext.Response.Responses = transactionProposalResponses
 	if len(transactionProposalResponses) > 0 {
-		requestContext.Response.Payload = transactionProposalResponses[0].ProposalResponse.GetResponse().Payload
+		responsePayload, err := getResultFromProposalResponse(transactionProposalResponses[0].ProposalResponse)
+		if err != nil {
+			requestContext.Error = err
```

Comment:
Is just returning the error OK or does it need to be wrapped as an SDK status error?

Reply:
@troyronda Can you confirm the desired behaviour (see comment above)?